### PR TITLE
Add example templates

### PR DESCRIPTION
**Pull request by dwightjl**
Templates to be used for creating new documents on docs.chain.link.

### CODE REVIEW

File: templates/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+   Expected results
+   ```
+
+1. {/* Step 2 */}
+
+   1. {/* Sub-step A */}
+   1. {/* Sub-step B */}
+
+1. {/* Step 3 */}
+
```
Comment (thedriftofwords) [inline templates/getting-started.mdx:76]: Do you think we should include an example of a Markdown table? (I'll admit, I just really like tables.)

**Review by thedriftofwords — COMMENTED** [review]
Also, add a standard disclaimer to the template ([example](https://github.com/smartcontractkit/quickstarts-lootbox#references)).

File: templates/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+{/\*
+Fill out the fields in the frontmatter above.
+
+- Section: What section of the top-nav this page appears in gettingStarted, dataFeeds, vrf, automation, chainlinkFunctions, anyApi, nodeOperator
+- Title: The title appearing at the top of the page
+- Whats Next: A list of pages to include in the What's Next section. Preferably imperative sentences describing the goal of the next page.
+  \*/}
+
+{/* Imports if necessary */}
+import { Aside } from "@components"
```
Comment (ritika-b) [inline templates/getting-started.mdx:17]:
```suggestion
import { Aside } from "@components"
import { YouTube } from "@astro-community/astro-embed-youtube"
```

**Review by ritika-b — CHANGES_REQUESTED** [review]

File: templates/getting-started.mdx (hunk 2 above)
Comment (ritika-b) [reply templates/getting-started.mdx:17]: Added this jic YouTube videos are also part of the tutorial

**Review by aelmanaa — APPROVED** [review]

File: templates/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+title: "Page Title"
+whatsnext: { "Try out the advanced tutorial": "/product-path/tutorials/advanced" }
+---
+
+{/\*
+Fill out the fields in the frontmatter above.
+
+- Section: What section of the top-nav this page appears in gettingStarted, dataFeeds, vrf, automation, chainlinkFunctions, anyApi, nodeOperator
+- Title: The title appearing at the top of the page
+- Whats Next: A list of pages to include in the What's Next section. Preferably imperative sentences describing the goal of the next page.
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply templates/getting-started.mdx:13]: Applied here: https://github.com/smartcontractkit/documentation/pull/1425/files

File: templates/getting-started.mdx (hunk 1 above)
Comment (dwightjl) [reply templates/getting-started.mdx:77]: Applied here: https://github.com/smartcontractkit/documentation/pull/1425/files

**Pull request merged**